### DISCUSSION TITLE
how can i make this work....

### DISCUSSION THREAD

**Discussion by eightsheep67** [jlesage/docker-handbrake#293] [General]
i have a little machine with n4120 and 8GB ram, i installed pve 8.0.4 and virtualize an openwrt as my main router os.
Then i want something more. 
i installed docker on ubuntu 22.04 with LXC, and i build a container of handbrake.
at the begining it works but cpu usage is full and speed is low. i know it didn't use the gpu
then I got the /dev/dri work, and preset shows intel qsv h265, but it just failed to transcode.
then i found a page says that insert "lowpower=0" into the preset can get this work, so i tried.
but when i put video file into /watch then start the container, my network lost and cannot connect to openwrt, pve, docker...nothing!
my machine seem like dead so i have to restart the machine, and tried it again. same thing happen. and i don't know the problem is.
i need somehelp, pls

**Comment by eightsheep67** [comment]
i see the /output there is a new folder means it started trascoding, but somehow it just dead.

here is my conversion.log

------- CONVERSION OUTPUT Tue Oct  3 10:15:25 UTC 2023 -------
[10:15:25] Compile-time hardening features are enabled
[10:15:25] hb_display_init: attempting VA driver 'iHD'
libva info: VA-API version 1.18.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /opt/intel/mediasdk/lib/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_18
libva info: va_openDriver() returns 0
[10:15:25] hb_display_init: using VA driver 'iHD'
libva info: VA-API version 1.18.0
libva info: User environment variable requested driver 'iHD'
libva info: Trying to open /opt/intel/mediasdk/lib/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_18
libva info: va_openDriver() returns 0
[10:15:26] qsv: is available on this system
Cannot load libnvidia-encode.so.1
[10:15:26] hb_init: starting libhb thread
[10:15:26] thread 7f6266f6fb38 started ("libhb")
HandBrake 1.6.1 (2023060800) - Linux x86_64 - https://handbrake.fr
2 CPUs detected
Opening /watch/input.mp4...
[10:15:26] CPU: Intel(R) Celeron(R) N4120 CPU @ 1.10GHz
[10:15:26]  - logical processor count: 2
[10:15:26] Intel Quick Sync Video support: yes
[10:15:26] Intel Quick Sync Video integrated adapter with index 0
[10:15:26] Impl mfxhw64 library path: /opt/intel/mediasdk/lib/libmfxhw64.so.1.35
[10:15:26]  - Intel Media SDK hardware: API 1.35 (minimum: 1.3)
[10:15:26]  - Decode support:  h264 hevc (8bit: yes, 10bit: yes) av1 (8bit: yes, 10bit: yes)
[10:15:26]  - H.264 encoder: yes
[10:15:26]     - preferred implementation: hardware (any) via ANY
[10:15:26]     - capabilities (hardware):  lowpower breftype icq+la+i+downs vsinfo chromalocinfo opt1 opt2+mbbrc+extbrc+trellis+repeatpps+ib_adapt+nmpslice
[10:15:26]  - H.265 encoder: yes (8bit: yes, 10bit: yes)
[10:15:26]     - preferred implementation: hardware (any) via ANY
[10:15:26]     - capabilities (hardware):  lowpower bpyramid icq vsinfo masteringinfo cllinfo opt1
[10:15:26]  - AV1 encoder: no
[10:15:26] hb_scan: path=/watch/input.mp4, title_index=1
udfread ERROR: ECMA 167 Volume Recognition failed
disc.c:333: failed opening UDF image /watch/input.mp4
disc.c:437: error opening file BDMV/index.bdmv
disc.c:437: error opening file BDMV/BACKUP/index.bdmv
[10:15:26] bd: not a bd - trying as a stream/file instead
libdvdread: DVDOpenFileUDF:UDFFindFile /VIDEO_TS/VIDEO_TS.IFO failed
libdvdnav: vm: vm: failed to read VIDEO_TS.IFO
[10:15:26] dvd: not a dvd - trying as a stream/file instead
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '/watch/input.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf58.27.100
  Duration: 03:33:10.21, start: 0.000000, bitrate: 5243 kb/s
  Stream #0:0[0x1](und): Video: h264 (High) (avc1 / 0x31637661), yuv420p(progressive), 1920x1080, 5103 kb/s, 29.97 fps, 29.97 tbr, 90k tbn (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : [0][0][0][0]
  Stream #0:1[0x2](eng): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 128 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]
[10:15:26] scan: decoding previews for title 1

Scanning title 1 of 1, preview 1, 0.00 %[10:15:26] scan: audio 0x1: aac, rate=48000Hz, bitrate=128510 English (AAC LC) (2.0 ch) (128 kbps)

Scanning title 1 of 1, preview 1, 10.00 %
Scanning title 1 of 1, preview 4, 40.00 %
Scanning title 1 of 1, preview 8, 80.00 %[10:15:27] scan: 10 previews, 1920x1080, 29.970 fps, autocrop = 0/0/0/0, aspect 16:9, PAR 1:1, color profile: 1-1-1, chroma location: left
[10:15:27] scan: supported video decoders: avcodec qsv
[10:15:27] libhb: scan thread found 1 valid title(s)
+ Using preset: 111111
+ title 1:
  + stream: /watch/input.mp4
  + duration: 03:33:10
  + size: 1920x1080, pixel aspect: 1/1, display aspect: 1.78, 29.970 fps
  + autocrop: 0/0/0/0
  + chapters:
    + 1: duration 03:33:10
  + audio tracks:
    + 1, English (AAC LC) (2.0 ch) (128 kbps) (iso639-2: eng)
  + subtitle tracks:
[10:15:27] Starting work at: Tue Oct  3 1

**Comment by eightsheep67** [comment]
here is some pve syslog before i reboot it

Oct 03 18:03:16 eightsheep pvedaemon[964]: <root@pam> successful auth for user 'root@pam'
Oct 03 18:04:46 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 18:04:46 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 18:04:46 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 18:04:46 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 18:04:46 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 18:05:00 eightsheep kernel: i915 0000:00:02.0: [drm] Resetting vcs0 for CS error
Oct 03 18:05:00 eightsheep kernel: i915 0000:00:02.0: [drm] HandBrakeCLI[130049] context reset due to GPU hang
Oct 03 18:05:00 eightsheep kernel: i915 0000:00:02.0: [drm] GPU HANG: ecode 9:4:cab6fff5, in HandBrakeCLI [130049]
Oct 03 18:07:11 eightsheep chronyd[761]: Selected source 44.190.40.123 (2.debian.pool.ntp.org)
-- Reboot --

Oct 03 23:20:35 eightsheep kernel: FS-Cache: Loaded
Oct 03 23:20:35 eightsheep kernel: Key type cifs.spnego registered
Oct 03 23:20:35 eightsheep kernel: Key type cifs.idmap registered
Oct 03 23:20:35 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 23:20:36 eightsheep kernel: docker0: port 2(veth116cb3b) entered blocking state
Oct 03 23:20:36 eightsheep kernel: docker0: port 2(veth116cb3b) entered disabled state
Oct 03 23:20:36 eightsheep kernel: device veth116cb3b entered promiscuous mode
Oct 03 23:20:36 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 23:20:36 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 23:20:36 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 23:20:36 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 23:20:36 eightsheep kernel: CIFS: Attempting to mount \\192.168.1.100\documents
Oct 03 23:20:36 eightsheep kernel: eth0: renamed from veth7d473c0
Oct 03 23:20:36 eightsheep kernel: IPv6: ADDRCONF(NETDEV_CHANGE): veth116cb3b: link becomes ready
Oct 03 23:20:36 eightsheep kernel: docker0: port 2(veth116cb3b) entered blocking state
Oct 03 23:20:36 eightsheep kernel: docker0: port 2(veth116cb3b) entered forwarding state
-- Reboot --

**Comment by jlesage** [comment]
A quick search of `[drm] GPU HANG` seems to show that this issue goes beyond Handbrake...

**Reply by eightsheep67** [reply]
now the main issue is Failure to initialise thread 'Quick Sync Video encoder (Intel Media SDK)
may you see #294
